packages/graphhopper/graphhopper.test.ts

```diff
@@ -1,4 +1,8 @@
 import { GraphHopper } from "./index"
+import dotenv from 'dotenv'
+if(process.env.NODE_ENV !== "production"){
+    dotenv.config()
+}
```

Comment:
Why do we need to check `NODE_ENV` here?

Reply:
It's just a good practice that I used to follow when dealing with env files. Like when you deploy your app it won't try to look into any local env files. I guess we don't need it as such, I just added it out of habit.